e2e/cypress/integration/ui/compliance/01_ssh_scan_job.spec.ts

```diff
@@ -18,100 +18,81 @@ describe('create a manual node ssh scan job and cleanup after', () => {
   const decoded = Base64.decode(Cypress.env('AUTOMATE_ACCEPTANCE_TARGET_KEY'));
 
   it('can create a credential with a username and key', () => {
-    // navigate to credentials create page:
-    // click on settings
-    cy.get('.nav-link').contains('Settings').click();
-
     // we save the route that will be called when we navigate to the page
     // in order to be able to wait for it later
     cy.route('POST', '/api/v0/secrets/search').as('getSecrets');
 
-    // click on node credentials in sidebar
-    cy.get('chef-sidebar-entry').contains('Node Credentials').click();
-    cy.url().should('include', '/settings/node-credentials');
-
-    // wait for data to return
-    cy.wait('@getSecrets');
-
     // click on add credential button to open create form
-    cy.contains('Add Credential').click().then(() => {
-      cy.url().should('include', '/settings/node-credentials/add');
+    cy.contains('Add Credential').click();
```

Comment:
using an empty `then()` is an anti-pattern that shouldn't be necessary since all Cypress commands are run in sequence. `then()` is useful for when we need to do something with the element returned by `get` (per [this doc](https://docs.cypress.io/api/commands/then.html#Syntax))